src/core/column-list/column.list.generate.js

```diff
@@ -1,15 +1,16 @@
 import {startCase} from '../utility';
 import {compile, getType} from '../services';
 
-export function generate(rows, columnFactory, deep = true) {
-	if (!rows || rows.length === 0) {
+export function generate(context) {
+	const deep = context.deep || true;
+	if (!context.rows || context.rows.length === 0 || !context.columnFactory) {
```

Comment:
title by default shouldBe startCase
signature is: title(propertyPath, graph, index)